src/main/resources/data.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE TASK
```

Comment:
Uruchomienie tego skryptu nie ma niestety żadnego efektu po uruchomieniu aplikacji :( Dzieje się tak dlatego, że masz dorzuconą opcję `spring.jpa.hibernate.ddl-auto=create`, co oznacza, że hibernate dropuje bazę danych i tworzy puste tabele od nowa. Zamień `create` na `validate`. Sprawi to, że hibernate spawdzi tylko czy potrzebne tabele istnieją ;) Btw zobaczysz wtedy komunikat, że brakuje cczegoś w deklaracji tej tabeli.